### PR TITLE
Escape character size fix

### PR DESCRIPTION
**Pull request by JensOgorek**
This contains the escape character fix for #3

### CODE REVIEW

File: src/x-tokenize.h [hunk 1]
```diff
@@ -398,23 +398,24 @@ owl_default_tokenizer_advance(struct owl_default_tokenizer *tokenizer,
             const char *string = text + content_offset;
             size_t string_length = content_length;
             if (has_escapes) {
+                // Issue: Zauberzeug/lizard#112 - Escape sequences are not applied correctly
```
Comment (falkoschindler) [inline src/x-tokenize.h:401]:
```suggestion
                // https://github.com/zauberzeug/lizard/pull/112 - Escape sequences are not applied correctly
```

File: diff_list.md [hunk 2]
```diff
@@ -5,3 +5,6 @@ Changes in this fork
 
 owl_example_parser.sh:
     - Add an example script to generate the parser from the grammar
+
+src/x-tokenize.h:
+    - Apply escape sequences correctly (Zauberzeug issue #112)
```
Comment (falkoschindler) [inline diff_list.md:10]:
```suggestion
    - Apply escape sequences correctly (https://github.com/zauberzeug/lizard/pull/112)
```

**Review by falkoschindler — CHANGES_REQUESTED** [review]
Let's use links when referring to the Lizard repo.

File: diff_list.md (hunk 2 above)
Comment (JensOgorek) [reply diff_list.md]: updated

**Review by falkoschindler — APPROVED** [review]

**Comment by falkoschindler** [comment]
@JensOgorek I think your merge commit reverted the -x flag for the generator script as well as the note in diff_list.md.

**Comment by JensOgorek** [comment]
> @JensOgorek I think your merge commit reverted the -x flag for the generator script as well as the note in diff_list.md.

I will fix what was missing on main after the merges. 
The 
```
main < a
             |
             +--- b
             |
             +--- c
             |
             +--- d
```
setup is not the easiest way to handle this

**Pull request merged**